library/src/main/java/com/heinrichreimersoftware/materialintro/slide/SimpleSlide.java

```diff
@@ -34,7 +35,7 @@
     private static final int DEFAULT_PERMISSIONS_REQUEST_CODE = 34; //Random number
     private SimpleSlideFragment fragment;
 
-
+    private final int id;
```

Comment:
Normally IDs are saved as `long` on Android. Not sure if that would make a huge difference, but if it's not too much effort for you, please change it.